src/core_systems/continuous_time_ode.jl

```diff
@@ -143,6 +143,7 @@ initial_state(integ::DEIntegrator) = integ.sol.prob.u0
 current_state(integ::DEIntegrator) = integ.u
 current_time(integ::DEIntegrator) = integ.t
 initial_time(integ::DEIntegrator) = integ.sol.prob.tspan[1]
+successful_step(integ::DEIntegrator) = successful_retcode(check_error(integ))
```

Comment:
For clarity, I'd do `using SciMLBase: successful_retcode, check_error` before this new function, so that we know where these names are coming from.